prac_06/programming_language.py

```diff
@@ -0,0 +1,16 @@
+class ProgrammingLanguage:
+    """represent type of code"""
```

Comment:
I would use an imperative voice and put full-stop at the end of the docstring as taught in lectures. `"""Represent type of code."""`
So does for line 4,  11, and 15 :)